From 68b83da896cbc370220725250a76cf8c6ac2d7d6 Mon Sep 17 00:00:00 2001
From: sergii-ssh <83605538+sergii-ssh@users.noreply.github.com>
Date: Thu, 25 Nov 2021 06:18:00 -0500
Subject: [release-1.12] Fix docker.save tar targets (#36086)

* Remove -default rather then DEFAULT_DISTRIBUTION (#36041)

* Re-trigger tests. Empty commit
---
 tools/istio-docker.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/istio-docker.mk b/tools/istio-docker.mk
index 6f67905211..c83482bb93 100644
--- a/tools/istio-docker.mk
+++ b/tools/istio-docker.mk
@@ -338,7 +338,7 @@ docker.all: $(DOCKER_TARGETS)
 DOCKER_TAR_TARGETS:=
 $(foreach TGT,$(DOCKER_TARGETS),$(eval tar.$(TGT): $(TGT) | $(ISTIO_DOCKER_TAR) ; \
          $(foreach VARIANT,$(DOCKER_BUILD_VARIANTS) default, time ( \
-		     docker save -o ${ISTIO_DOCKER_TAR}/$(subst docker.,,$(TGT))$(call variant-tag,$(VARIANT)).tar $(HUB)/$(subst docker.,,$(TGT)):$(subst -$(DEFAULT_DISTRIBUTION),,$(TAG)-$(VARIANT)) && \
+		     docker save -o ${ISTIO_DOCKER_TAR}/$(subst docker.,,$(TGT))$(call variant-tag,$(VARIANT)).tar $(HUB)/$(subst docker.,,$(TGT)):$(subst -default,,$(TAG)-$(VARIANT)) && \
              gzip -f ${ISTIO_DOCKER_TAR}/$(subst docker.,,$(TGT))$(call variant-tag,$(VARIANT)).tar \
 			   ); \
 		  )))
-- 
2.35.3

